WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewController.swift

```diff
@@ -17,8 +17,11 @@ final class ShippingLabelAddressFormViewController: UIViewController {
 
     /// Top banner that shows a warning in case there is an error in the address validation.
     ///
-    private lazy var topBannerView: TopBannerView = {
-        let topBanner = ShippingLabelAddressTopBannerFactory.addressErrorTopBannerView(shipType: viewModel.type) { [weak self] in
+    private var topBannerView: TopBannerView {
```

Comment:
I really appreciate that you thought of the case where a phone number is added to the form. 🎉   However, an unfortunate side effect of making this a computed property instead of lazy is that multiple instances of the banner can appear on the screen.

For example, if you start with an invalid address on the order and no phone number, you'll see the banner with just the "Open Map" button. If you then add a phone number and tap "Done" you'll see a second banner with the "Contact Customer" button. Each time you tap "Done" another banner appears:

Banner is initially shown | After adding phone and tapping "Done"
-|-
![Simulator Screen Shot - iPhone 11 Pro - 2021-07-21 at 18 36 27](https://user-images.githubusercontent.com/8658164/126537023-afee99f6-db9a-435a-b615-a4bffcbccfa5.png)|![Simulator Screen Shot - iPhone 11 Pro - 2021-07-21 at 18 36 38](https://user-images.githubusercontent.com/8658164/126537034-18198323-71a3-402c-9e87-4eaf5a5f8f78.png)

Reply:
Thank you for catching that bug! I just pushed a fix for this in db7ad1a4028024ddf8620de238c7f349b18c9d38 by removing the old banner from the stack view when displaying the banner again.